### PR TITLE
Experimental: rewrite of the architecture section

### PR DESCRIPTION
**Pull request by Dfinity-Bjoern**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: how-it-works/00_General/01_overview-of-the-internet-computer.card.md [hunk 1]
```diff
@@ -6,36 +6,31 @@ title: Architecture of the Internet Computer
...
+To achieve this, blockchain technology – the foundation of the IC – had to be reconceived from ground up, building on many lessons learned from earlier blockchain architectures.
 
-Smart contracts on the Internet Computer are called *canister smart contracts*, or *canisters*, each consisting of a bundle of [*WebAssembly (Wasm)*](https://en.wikipedia.org/wiki/WebAssembly) bytecode and smart contract data storage.
-Each canister has its own, isolated, data storage that is only changed when the canister executes code.
+The Internet Computer hosts smart contracts in the format of *canisters*, each consisting of a [*WebAssembly (Wasm)*](https://en.wikipedia.org/wiki/WebAssembly) bytecode module and data storage.
+Canisters execute concurrently and can call each other via an asynchronous inter-canister messenging protocol. They have access to advanced functionality including secure randomness, ECDSA signatures, or HTTPS outcalls, and can serve web content directly into the browser.
```
Comment (victorshoup-dfinity) [inline how-it-works/00_General/01_overview-of-the-internet-computer.card.md:13]: messenging/messaging

File: how-it-works/00_General/01_overview-of-the-internet-computer.card.md [hunk 2]
```diff
@@ -6,36 +6,31 @@ title: Architecture of the Internet Computer
...
-A subnet is an independent blockchain, running on *node machines*, or *nodes*, deployed in globally-distributed data centers.
+The *Internet Computer Protocol (ICP)* is based on a high-throughput, low-latency consensus mechanism and an efficient virtual machine for WebAssembly execution.
+The central innovation of ICP is [*chain-key cryptography*](https://internetcomputer.org/how-it-works/#Chain-key-cryptography), a suite of advanced cryptographic protocols based on [threshold cryptography](https://en.wikipedia.org/wiki/Threshold_cryptosystem) that enables the decentralized operation of the IC with unprecedented scalability.
+The Internet Computer also implements [*chain-key transactions*](https://internetcomputer.org/how-it-works/#Chain-key-transactions), which enable a canister to interact with (write to) other blockchains using threshold cryptography.
+ICP also includes a sophisticated collection of technologies for robustly and securely addressing operational concerns, such as how to deal with faulty nodes or protocol upgrades, which we call [*chain-evolution technology*](https://internetcomputer.org/how-it-works/#Chain-evolution-technology).
```
Comment (victorshoup-dfinity) [inline how-it-works/00_General/01_overview-of-the-internet-computer.card.md:19]: POSSIBLY: faulty nodes, protocol upgrades, or subnet membership changes

Comment (victorshoup-dfinity) [reply how-it-works/00_General/01_overview-of-the-internet-computer.card.md:19]: Oops..I see we have not yet introduced subnet, so please ignore

File: how-it-works/00_General/01_overview-of-the-internet-computer.card.md (hunk 1 above)
Comment (victorshoup-dfinity) [inline how-it-works/00_General/01_overview-of-the-internet-computer.card.md:13]: ECDSA signatures *and* HPPTS outcalls

Also: can we use a less technical / more descriptive term than "HTTPS outcalls"?

File: how-it-works/00_General/01_overview-of-the-internet-computer.card.md [hunk 3]
```diff
@@ -6,36 +6,31 @@ title: Architecture of the Internet Computer
...
-To achieve this, blockchain technology – the foundation of the IC – had to be reconceived from ground up, building on many lessons learnt from earlier blockchain projects.
+The Internet Computer (IC) realizes the vision of a *World Computer* – an open and secure blockchain-based network that hosts programs and data in the form of smart contracts and scales to unbounded capacity.
+To achieve this, blockchain technology – the foundation of the IC – had to be reconceived from ground up, building on many lessons learned from earlier blockchain architectures.
 
-Smart contracts on the Internet Computer are called *canister smart contracts*, or *canisters*, each consisting of a bundle of [*WebAssembly (Wasm)*](https://en.wikipedia.org/wiki/WebAssembly) bytecode and smart contract data storage.
-Each canister has its own, isolated, data storage that is only changed when the canister executes code.
+The Internet Computer hosts smart contracts in the format of *canisters*, each consisting of a [*WebAssembly (Wasm)*](https://en.wikipedia.org/wiki/WebAssembly) bytecode module and data storage.
```
Comment (victorshoup-dfinity) [inline how-it-works/00_General/01_overview-of-the-internet-computer.card.md:12]: I would prefer: The Internet Computer represents smart contracts in the form of *canisters*, ...

Why? We already say the IC *hosts* programs, and we say these programs are smart contracts, so we are re-suing the term "host" too mnay times (and are saying a smart contracts hosts canisters, if you expand the macros).
Also, I prefer "in the form" to the ueber-technical "in the format"

**Comment by JensGroth** [comment]
Meta: do we expect readers to be from the blockchain space? The text implicitly assumes knowledge of the following terminology: blockchain, smart contract, consensus, sharding, dapp, tokenized, dao, decentralization.

**Comment by jessiemongeon1** [comment]
Outdated, confirmed with Björn via Slack this can be closed.

**Pull request closed without merging**